english/marshmallow.md

```diff
@@ -966,6 +966,7 @@ The lannisters always pays thier debts, says tyrion lannister. Interesting [char
 
 Time to [move to Canada](Canada/Canada.md), eh?
 
+[Click](mmacoder/mmacoder.md) and follow my [blog](http://www.mmacoder.com) into MMA and app development stardom!
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.